Library/Formula/eg.rb

```diff
@@ -0,0 +1,83 @@
+class Eg < Formula
+  desc "Expert Guide. Norton Guide Reader For GNU/Linux"
+  homepage "http://www.davep.org/norton-guides/"
+  url "http://www.davep.org/norton-guides/eg-1.00.tar.gz"
+  sha256 "e985b2abd160c5f65bea661de800f0a83f0bfbaca54e5cbdc2e738dfbbdb164e"
+
+  depends_on "s-lang"
+
+  patch :DATA
```

Comment:
Need an upstream submission to link to here and a wee explanation of what is being fixed, Thanks!